features/steps/document.js

```diff
@@ -265,10 +263,8 @@ When('I update the documents [{string}, {string}]', async function (id1, id2) {
 
 
 Then('I get an error with message {string}', function (message) {
-  should(this.error)
-    .not.be.null();
-  should(this.error.message)
-    .eql(message);
+  should(this.error).not.be.null();
+  should(this.error.message).startWith(message);
```

Comment:
This is the actual functional tests fix.

Reply:
As the error message change will impact all sdks, I would rather have updated the submodule reference (the error message is already modfiied on the HEAD of `sdk-features`)